This test in 1.14.x branch is time-sensitive, only 1.15.x
has mocked time to guarantee this test works without having
to rely on the system clock

not applicable to upstream

diff --git a/integrations/api_user_heatmap_test.go b/integrations/api_user_heatmap_test.go
index 105d39e..aeb8950 100644
--- a/integrations/api_user_heatmap_test.go
+++ b/integrations/api_user_heatmap_test.go
@@ -15,6 +15,11 @@ import (
 )
 
 func TestUserHeatmap(t *testing.T) {
+	// This test in 1.14.x branch is time-sensitive, only 1.15.x
+	// has mocked time to guarantee this test works without having
+	// to rely on the system clock
+	t.Skip("sensitive to system clock")
+
 	defer prepareTestEnv(t)()
 	adminUsername := "user1"
 	normalUsername := "user2"
diff --git a/models/user_heatmap_test.go b/models/user_heatmap_test.go
index cc08ac1..581d67e 100644
--- a/models/user_heatmap_test.go
+++ b/models/user_heatmap_test.go
@@ -13,6 +13,11 @@ import (
 )
 
 func TestGetUserHeatmapDataByUser(t *testing.T) {
+	// This test in 1.14.x branch is time-sensitive, only 1.15.x
+	// has mocked time to guarantee this test works without having
+	// to rely on the system clock
+	t.Skip("sensitive to system clock")
+
 	testCases := []struct {
 		userID      int64
 		doerID      int64
